src/main/webapp/web.xml

```diff
@@ -3,106 +3,7 @@
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>deleteDriver</servlet-name>
-        <servlet-class>mate.controller.driver.DeleteDriverController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>deleteDriver</servlet-name>
-        <url-pattern>/drivers/delete</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addDriver</servlet-name>
-        <servlet-class>mate.controller.driver.AddDriverController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addDriver</servlet-name>
-        <url-pattern>/drivers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>allCars</servlet-name>
-        <servlet-class>mate.controller.car.GetAllCarsController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>allCars</servlet-name>
-        <url-pattern>/cars/</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addCar</servlet-name>
-        <servlet-class>mate.controller.car.AddCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addCar</servlet-name>
-        <url-pattern>/cars/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addDriverToCar</servlet-name>
-        <servlet-class>mate.controller.car.AddDriverToCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addDriverToCar</servlet-name>
-        <url-pattern>/cars/drivers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>deleteCar</servlet-name>
-        <servlet-class>mate.controller.car.DeleteCarController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>deleteCar</servlet-name>
-        <url-pattern>/cars/delete</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>allManufacturers</servlet-name>
-        <servlet-class>mate.controller.manufacturer.GetAllManufacturersController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>allManufacturers</servlet-name>
-        <url-pattern>/manufacturers/</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>addManufacturer</servlet-name>
-        <servlet-class>mate.controller.manufacturer.AddManufacturerController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>addManufacturer</servlet-name>
-        <url-pattern>/manufacturers/add</url-pattern>
-    </servlet-mapping>
-
-    <servlet>
-        <servlet-name>deleteManufacturer</servlet-name>
-        <servlet-class>mate.controller.manufacturer.DeleteManufacturerController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>deleteManufacturer</servlet-name>
-        <url-pattern>/manufacturers/delete</url-pattern>
-    </servlet-mapping>
+    <session-config>
+        <session-timeout>30</session-timeout>
+    </session-config>
```

Comment:
did you need to add this configuration according to the task's description? Also, what is the default value for the session's timeout?

Reply:
Just for test, there are a lot of things were added for test. I believe it's 30 minutes by default.

Reply:
so, remove it)